gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.5.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.8.2-bin.zip
```

Comment:
@TobiGr should gradle be updated?

Reply:
It works fine in Intellij, NewPipe uses 6.5 so I thought why not.

Reply:
Grade should be updated separately. Ideally the version used in the extractor and app are the same.